algorithms/C/README.md

```diff
@@ -22,3 +22,9 @@
 - [Count Words](strings/count-words.c)
 - [Palindrome](strings/palindrome.c)
 - [Permutation of String](string/Permutation-of-String.c)
+
+## Searching
+
+* [Binary Search](searching/Binary-search.c)

```

Comment:
Correct the format. Write it like the rest.

Reply:
done